cross/fish/Makefile

```diff
@@ -9,17 +9,13 @@ DEPENDS = cross/ncursesw cross/pcre2
 
 HOMEPAGE = https://fishshell.com
 COMMENT = fish is a smart and user-friendly command line shell for Linux, macOS, and the rest of the family.
-LICENSE = Custom
+LICENSE = GPL-2.0
 
 UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(OLD_PPC_ARCHS)
 
 include ../../mk/spksrc.cross-cmake.mk
 
 CMAKE_ARGS += -DCURSES_NCURSES_LIBRARY=$(INSTALL_DIR)$(INSTALL_PREFIX)/lib/libncursesw.so
+CMAKE_ARGS += -DCURSES_NCURSES_LIBRARY=$(INSTALL_DIR)$(INSTALL_PREFIX)/lib/libtinfow.so
+CMAKE_ARGS += -DCURSES_EXTRA_LIBRARY=$(INSTALL_DIR)$(INSTALL_PREFIX)/lib/libtinfow.so
```

Comment:
I had to apply a similar fix to cross/mtr recently (5280921b92593d0440b407b5756b5323397216c6)

Reply:
Thanks @hgy59

Reply:
@publicarray this is a mess. Some more tools fail to find this dependency when ncurses is built with `--with-termlib`.

I will make `--with-termlib` optional, as it is only required to build `lsblk` tool for synocli-misc (cross/misc-util-linux) introduced in #5748.

Found this while updating irssi (requested in https://github.com/SynoCommunity/spksrc/pull/4564#issuecomment-1595419260 and #5237)